docs/software-platforms/lsfr/creating-vcs.md

```diff
@@ -0,0 +1,104 @@
+        - If you are planning to develop in R, we recommend choosing `Rstudio`, which includes RStudio.
+        - If you are planning to develop in Python, we recommend choosing `VSCodium`, which includes VSCodium, a distribution of Microsoft's editor VS Code.
+        - For information on all application options, see [the LSfR documentation on Applications](https://docs.aws.amazon.com/lightsail-for-research/latest/ug/blueprints-plans.html).
+    - Choose any of the availability zones.
+    - Name your instance, be sure to make sure this is different than the name chose for your product.
+    - Pick the size for your instance.
+        - Use this [table describing the total vCPUs, memory, and storage space](https://docs.aws.amazon.com/lightsail-for-research/latest/ug/blueprints-plans.html#plans) to choose the most appropriate instance plan.
+        - Before you request a plan with GPUs, you will need to put in a [request with the Data Lab team](../../getting-started/accessing-resources/getting-access-to-compute.md#gpu-instance-access).
+    - Set the `ShutdownIdlePercent` to 5 and the `ShutdownTimePeriod` to 10.
+    This means if the instance is using less than 5% of the total available CPUs or inactive for more than 10 minutes, the instance will temporarily shut down.
+    This _does not_ terminate your instance, it temporarily stops any idle instances until you are ready to resume your work.
+
+    <figure markdown="span">
+        ![Configure instance](../../img/creating-vcs-4.png){width="600"}
+    </figure>
+
+1. Once you have configured your instance, click `Launch product`.
+You have now created a virtual computer!
+
+## How to access a virtual computer
+
+Once you have created your virtual computer, you will be able to access and launch the computer on a web browser.
+Note that it can take 5-10 minutes to create the instance.
+
+1. To access a virtual computer, you will need to navigate to Lightsail for Research.
+Search for Lightsail for Research using the search bar and then select the product.
+
+    <figure markdown="span">
+        ![Search for Lightsail](../../img/creating-vcs-5.png){width="600"}
+    </figure>
+
+1. This will take you directly to a page that lists your virtual computers.
+You should see the virtual computer you created with the instance name that you provided during set up.
+
+    <figure markdown="span">
+        ![List of computers](../../img/creating-vcs-6.png){width="600"}
+    </figure>
+
+1. To launch the computer, click `Start computer`.
+When the computer is ready, use the drop-down menu in the lower right-hand side to select `Access operating system`.
+
+    <figure markdown="span">
+        ![Start computer](../../img/creating-vcs-7.png){width="600"}
+    </figure>
+
+    You can also use `Launch Rstudio` or `Launch VSCodium`, but if you do that you will have access to _only_ that application instead of the Ubuntu desktop and all installed applications.
```

Comment:
Something I experienced when opening RStudio is that I had to hit the `x` to close it before it pointed to 4.3.3. Might want to note that here? Or maybe we create a troubleshooting section for LfR this week?

Reply:
Hmm I was able to open Rstudio both from clicking `Launch Rstudio` and from opening the application on the desktop and R 4.3.3 was being used?

Reply:
I think it boils down to timing.